### ISSUE TITLE
iOS 13 users with Dark Mode enabled will experience incorrect label and background colors

### ISSUE THREAD

**Issue by kaylagalway** [mozilla-lockwise/lockwise-ios#1112]
## Steps to reproduce
- run updated app(branch ios-13-ui) on device with iOS 13
- enable dark mode
- open app

### Expected behavior
- All UI elements should display normally

### Actual behavior
- Unable to read most labels as color has been changed to white
- Some background colors are incorrect

### Device & build information
* Device: iPhone X
* Build version: branch ios-13-ui
### Notes
Attachments: 
![IMG_0346](https://user-images.githubusercontent.com/15383688/68260074-03f77780-fff9-11e9-95c1-40162953f5f1.PNG)
![IMG_0345](https://user-images.githubusercontent.com/15383688/68260079-0a85ef00-fff9-11e9-85b8-d7afd299d400.PNG)
![IMG_0342](https://user-images.githubusercontent.com/15383688/68260096-1671b100-fff9-11e9-8f17-cad98d173517.PNG)
![IMG_0343](https://user-images.githubusercontent.com/15383688/68260102-1b366500-fff9-11e9-919b-08c950a32162.PNG)

**Comment by kaylagalway** [comment]
Dark mode has been temporarily disabled on the "ios-13-ui" branch until the larger fixes can be made

**Comment by ZekeMedley** [comment]
This is happening some for me as well running version 1.6.2 on an iPhone 11. Here is an example from the homepage of the app. The top bar appears to be rendering incorrectly.

![image](https://user-images.githubusercontent.com/30676292/68515255-270e6b00-0235-11ea-94a4-09d2169ea23b.png)

**Comment by kaylagalway** [comment]
> This is happening some for me as well running version 1.6.2 on an iPhone 11. Here is an example from the homepage of the app. The top bar appears to be rendering incorrectly.
> 
> ![image](https://user-images.githubusercontent.com/30676292/68515255-270e6b00-0235-11ea-94a4-09d2169ea23b.png)

Yes, thank you. That issue was addressed in the most recently merged PR: https://github.com/mozilla-lockwise/lockwise-ios/pull/1106
